docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -0,0 +1,313 @@
+
+          <para>If you have added it, update to the latest charts:</para>
+
+          <para><programlisting>helm repo update</programlisting></para>
+        </listitem>
+
+        <listitem>
+          <para>Create a new text file and name it <emphasis
+          role="bold">twothors.yaml</emphasis> and open it in a text
+          editor.</para>
+
+          <para>You can use any text editor.</para>
+        </listitem>
+
+        <listitem>
+          <para>Open the default values file that you saved earlier
+          (myvalues.yaml) in a text editor.</para>
+        </listitem>
+
+        <listitem>
+          <para>Copy the entire <emphasis role="bold">thor:</emphasis> section
+          and paste it into the new twothors.yaml file.</para>
+        </listitem>
+
+        <listitem>
+          <para>Copy the entire contents, except the first line (thor:), and
+          paste it at the end of the file.</para>
+        </listitem>
+
+        <listitem>
+          <para>In the second block, edit the value for <emphasis
+          role="bold">name:</emphasis> and change it to <emphasis
+          role="bold">thor10.</emphasis></para>
+        </listitem>
+
+        <listitem>
+          <para>In the second block, edit the value for <emphasis
+          role="bold">prefix:</emphasis> and change it to <emphasis
+          role="bold">thor10.</emphasis></para>
+        </listitem>
+
+        <listitem>
+          <para>In the second block, edit the value for <emphasis
+          role="bold">numWorkers:</emphasis> and change it to <emphasis
+          role="bold">10.</emphasis></para>
+        </listitem>
+
+        <listitem>
+          <para>In the second block, edit the value for <emphasis
+          role="bold">maxJobs:</emphasis> and change it to <emphasis
+          role="bold">20.</emphasis></para>
+        </listitem>
+
+        <listitem>
+          <para>Save the file and close the text editor.</para>
+
+          <para>The resulting twothors.yaml file should look like this</para>
+
+          <para><emphasis role="bold">Note:</emphasis> The comments have been
+          removed to simplify the example:</para>
+
+          <para><programlisting>thor:
+- name: thor
+  prefix: thor
+  numWorkers: 2
+  maxJobs: 4
+  maxGraphs: 2
+- name: thor10
+  prefix: thor10
+  numWorkers: 10
+  maxJobs: 20
+  maxGraphs: 2</programlisting></para>
+        </listitem>
+      </orderedlist>
+
+      <para><emphasis role="bold">Deploy using the new custom configuration
+      chart.</emphasis></para>
+
+      <orderedlist>
+        <listitem>
+          <para>Open a terminal and navigate to the folder where you saved the
+          twothors.yaml file.</para>
+        </listitem>
+
+        <listitem>
+          <para>Deploy your HPCC Systems Platform, adding the new
+          configuration to your command:</para>
+
+          <para><programlisting>helm install mycluster hpcc/hpcc -f tworoxies.yaml -f twothors.yaml</programlisting></para>
```

Comment:
NB: this will fail, if they haven't at some point uninstalled the previous mycluster installation.
If they haven't, they have the option to upgraded (helm upgrade) instead...